docs/plugins.md

```diff
@@ -1042,6 +1042,7 @@ The following plugins provide new ways of displaying overlay data information.
 These plugins provide new markers or news ways of converting abstract data into images in your screen. Leaflet users versed in GIS also know these as symbolizers.
 
 <table class="plugins"><tr><th>Plugin</th><th>Description</th><th>Maintainer</th></tr>
+	<te
```

Comment:
Hi, please remove this typo.

Reply:
I just corrected it and thanks for your reminder.

Reply:
looks like it's still present

Reply:
Okay, I'll pull a new  request.Its my first time use Github,sorry about that